woocommerce/woocommerce-ios
# Order Creation: Fix some accessibility issues

## Description

This PR improves a few accessibility issues related to VoiceOver.

## Changes

- Adds accessibility labels to "Edit Status" and "Edit Customer Details" buttons.
  - closes https://github.com/woocommerce/woocommerce-ios/issues/6071.
- Adds accessibility labels for the back button in the variations list.
  - closes https://github.com/woocommerce/woocommerce-ios/issues/6072
- Adds accessibility properties to `TitleAndValueRow` to prevent "button" trait for non-selectable version.
  - closes https://github.com/woocommerce/woocommerce-ios/issues/6073
- Adds header traits to address form sections headers.
  - closes https://github.com/woocommerce/woocommerce-ios/issues/6074
- Improves search bar accessibility properties in `FilterListSelector`.
  - closes https://github.com/woocommerce/woocommerce-ios/issues/6075
  - closes https://github.com/woocommerce/woocommerce-ios/issues/6077

## Testing

You can test with the "Accessibility Inspector" app or by enabling VoiceOver on the device.

1. Build and run the app in debug mode (to ensure the feature flag is enabled).
2. Make sure Order Creation is enabled under Settings > Experimental Features.
3. Go to the Orders tab and tap the + button. Tap "Create order" in the bottom sheet.
4. Go over the list of changed elements from above and confirm that traits and labels are correct.

---
